### PR TITLE
NOSTORY: PR to add lb tlsv v1.2, update files

### PR DESCRIPTION
**Pull request by patrickmukumbu**
# Description

PR to add lb tlsv v1.2, update files

## Features/Fixes/Patches/Changes list:

Please check the [v1.1.4 CHANGELOG.md](https://github.com/boldlink/terraform-aws-ecs-service/blob/fixes/lb-tls-v12/CHANGELOG.md#114---2022-10-10)

### PR Owners Checklist:
* [ ] Have you followed the guidelines in our Contributing document?
* [ ] Have you checked to ensure there aren't other open [Pull Requests](../../../pulls) for the same update/change?
* [ ] Have you checked there aren't open [Pull Requests](../../../pulls) of upstream or parallel code that will cause conflicts?
* [ ] Have you updated the `CHANGELOG.md`?
* [ ] Have you updated the `README.md`(s)?
* [ ] OWNER: Does your submission pass?
    * [ ] Pre-commit (attach logs/print screen to this PR)
    * [ ] Checkov/terrascan (attach logs/print screen to this PR)
    * [ ] Terraform resources examples build/destroy successfully with Terraform 0.14.11 minimum?
* [ ] Have you added an explanation of what your changes do and why you'd like us to include them?
* [ ] Have you written new tests for your core changes, as applicable?
* [ ] Have you successfully run tests with your changes locally?
* [ ] First PR? Have you deleted `## How to use this template...` from the root `.README.md`?

### PR Reviewers Checklists?
* [ ] Are all your comments/changes resolved?
* [ ] Are you happy with the OWNER checklists and additional information provided?
* [ ] Does your review tests pass?
    * [ ] Terraform resources examples build/destroy successfully with Terraform 0.14.11 minimum?

### CODE REVIEW

**Review by ndegwajohn — CHANGES_REQUESTED** [review]
To reduce the number of VPCs created when testing examples, I suggest we have one as a supporting resource.

File: CHANGELOG.md [hunk 1]
```diff
@@ -20,6 +18,14 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - feat: use updated ecs cluster source for ecs cluster
 - feat: consolidate ecs cluster module and ecs service module into one
 
+## [1.1.4] - 2022-10-10
+### Changes
+- fix: CKV_AWS_103: "Ensure that load balancer is using TLS 1.2"
+- feat: update files
+- feat: update github workflows
+- feat: Add supporting resources (vpc and ecs cluster)
+- feat: Make creation of KMS Key optional (i.e external KMS key can be used)
```
Comment (ndegwajohn) [inline CHANGELOG.md:27]: hey @patrickmukumbu should we rename the branch name to have a feature prefix?

File: examples/complete/main.tf [hunk 2]
```diff
@@ -34,45 +7,21 @@ module "access_logs_bucket" {
...
   #checkov:skip=CKV_AWS_111:Ensure IAM policies does not allow write access without constraints"
   #checkov:skip=CKV_AWS_91:Ensure IAM policies does not allow write access without constraints"
-  #checkov:skip=CKV_AWS_103:Ensure IAM policies does not allow write access without constraints"
   requires_compatibilities = ["FARGATE"]
   network_mode             = "awsvpc"
   name                     = "${local.name}-service"
   family                   = "${local.name}-task-definition"
   network_configuration = {
-    subnets          = flatten(module.vpc.public_subnet_id)
+    subnets          = local.public_subnets
```
Comment (ndegwajohn) [inline examples/complete/main.tf:19]: Run the service in private subnets

**Review by ndegwajohn — CHANGES_REQUESTED** [review]
Please see my comments

File: CHANGELOG.md (hunk 1 above)
Comment (patrickmukumbu) [reply CHANGELOG.md:27]: Hey @ndegwajohn  since the scope was first to finish the fix I think it would just be great for it to remain that way

**Review by ndegwajohn — APPROVED** [review]
Tested and LGTM!

**Pull request merged**